book/website/collaboration/inclusive-events/inclusive-events-content-guide.md

```diff
@@ -0,0 +1,98 @@
+
+(cl-inclusive-events-content-accessibility)=
+## Content Accessibility Guidance 
+Ensuring that event communications, including printed content, are accessible to all attendees is critical.
+
+Please review the following recommendations and ensure that any material you have designed for your event meets the accessibility requirements of your atendees. 
+
+#### Use a size 14 (or larger) font for both digital and printed content.
+- A size 14 font is the recommended size to make text information legible for a broad audience and support people with lived experience of visual impairments, and other conditions that affect readability.
+
+#### Use a Sans Serif font such as Arial, Tahoma, Verdana, Helvetica or Monserrat.
+- "Sans Serif" refers to fonts that do not have embellishments or lines on the letters.
+- Using a *Serif* font such as Times New Roman can negatively impact readability and make it harder for people with dyslexia to read information. 
+- Using a *Sans Serif* font is especially important for online content such as emails and websites.
+
+#### Use high contrast colours in all content.
+- The best example of high contrast colours is using dark text on a light background.
+- If you are planning on using different colour palettes to create content and make communication more visually appealing, please ensure that your colours meet Accessibility Standards for Low Vision and Colour Blindness.
+- You may want to consider giving speakers and presenters a guide on how to make their posters and presentations accessible if your event hosts a poster session.
+#### Avoid backgrounds that have an excessive amount of colors, shapes, or graphics.
+- In this manner, reading becomes more enjoyable since the text won't be overwhelmed by an overly busy background.
+#### Do not embed essential information in an image.
+- This includes the information about the name, date, time and location of the event, and any accessibility information.
+- Images do not just mean images on a page, but also using a PNG or JPEG file type for flyers, presentation, or other content.
+- Images cannot be ‘read’ by screen readers that are being used by a person with a visual impairment.
+- Consider using plain text for emails and providing flyers or images as attachments that are accompanied by Alternative Text Descriptions. 
+
+### Provide Alt-text (Alternative Text) descriptions for online images in your content.
+- This is helpful for people with visual impairments that use screen readers, and is also helpful for people with low bandwidth internet where images may not load on a page.
+
+### Avoid blinking or flashing that could cause seizures. 
+- Some people find certain motions or animations to be highly distracting, and some people may even become sick from them. Therefore, consider if the motion is necessary and if it will make the material easier to understand.
```

Comment:
```suggestion
- Some people find certain motions or animations to be highly distracting, and some people may even become sick from them. 
Therefore, consider if the motion is necessary and if it will make the material easier to understand.
```